data/remnant/remnant 2 side missions.txt

```diff
@@ -441,7 +441,7 @@ mission "Remnant: Shattered Light 2"
 		conversation
 			`Your arrival in the spaceport finds Taely in deep conversation with an elderly Remnant, empty food trays cluttering the table before them. They both turn when you approach, breaking off their conversation.`
 			`	"Thank you for coming, Captain <last>. I would like to introduce you to Ruach. She was a shipyard engineering prefect for many decades, but now spends most of her time as a teacher."`
-			`	Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with that which the Embers blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
+			`	Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty making the best of whatever the Embers graciously blessed me with, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
```

Comment:
The core change here seems to be "that which" -> "whatever".
Moving "with" to the end of the clause seems like a regression in readability to me.
I'm also not sure why "graciously" is being added here. It, and all the other changes seem extraneous.
The simplest change would be:
```suggestion
			`	Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with what the Embers graciously blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
```
Or:
```suggestion
			`	Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with whatever the Embers graciously blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
```
But, "that which" is generally considered more formal than "what" or "whatever", so while the literal meaning of the words is unchanged, the characterisation they convey is, in my opinion, harmed.